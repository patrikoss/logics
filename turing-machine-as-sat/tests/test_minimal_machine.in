1 1 0
a
q
q
q
3
a a a
