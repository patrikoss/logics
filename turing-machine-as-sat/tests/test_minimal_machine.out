YES

a|q a|  a| 
